File: m.py
import requests

url = "https://barcodes-lookup.p.rapidapi.com/"

# querystring = {"query":"9780008108335"}
querystring = {"query":"6151100044739"}
# querystring = {"query":"9988012098"}

headers = {
	"x-rapidapi-key": "ee6b4943bcmsh514f1e849b5f265p127446jsnf0114804cfbf",
	"x-rapidapi-host": "barcodes-lookup.p.rapidapi.com"
}

response = requests.get(url, headers=headers, params=querystring)

print(response.json())

File: m.js
const http = require('https');

const options = {
	method: 'GET',
	hostname: 'barcodes-lookup.p.rapidapi.com',
	port: null,
	path: '/?query=9780439625593',
	headers: {
		'x-rapidapi-key': 'ee6b4943bcmsh514f1e849b5f265p127446jsnf0114804cfbf',
		'x-rapidapi-host': 'barcodes-lookup.p.rapidapi.com'
	}
};

const req = http.request(options, function (res) {
	const chunks = [];

	res.on('data', function (chunk) {
		chunks.push(chunk);
	});

	res.on('end', function () {
		const body = Buffer.concat(chunks);
		console.log(body.toString());
	});
});

req.end();